### DISCUSSION TITLE
Struggling to rebase

### DISCUSSION THREAD

**Discussion by aljawaid** [aljawaid/ColorManager#14] [Q&A]
@creecros  any tips on how to rebase a fork? My aljawaid/kanboard_website is forked but shows as 10 commits ahead. I followed so many tutorials on rebasing forks but I can't get it to work. I tried different branches and all of them are rebase as up to date but it still shows 10.

I know when I do another PR, Frédéric will comment again as originally it was around 92 commits ahead.

Any tips or commands?

I use GitExtensions (console for rebasing) and GitHub Web only

**Comment by creecros** [comment]
unfortunately, i have never used rebase, so no idea

**Reply by aljawaid** [reply]
OK so how do you create a linear history when you do PRs to kanboard/website?

**Reply by creecros** [reply]
dunno, i just do what i do and send a PR request. I don't believe ever ever had more than 1 commit tho, i can't imagine do more than 1 commit, why do you have 90 something commits? :)

**Reply by creecros** [reply]
oh....cause you use github web to make code edits, so litteraly every edit you make is a commit. I used to do that, back in the day, before i knew better easier ways.
